### PR TITLE
Hook in select

### PR DESCRIPTION
**Pull request by juwara0**
# feature

Closes: https://github.com/ciena-frost/ember-frost-core/issues/163
# CHANGELOG
- **Added** hooks for testing into frost-select and frost-multi-select
- **Added** tests to verify hooks in integration tests for frost-select and frost-multi-select
- **Added** documentation on the hook usage for frost-select and frost-multi-select

### CODE REVIEW

File: docs/frost-select.md [hunk 1]
```diff
@@ -18,6 +18,14 @@
 | `onInput`      | `string` | `<action-name>` | The action callback to call when the value of the filter changes as the user types |
 | `placeholder` | `string` | | Placeholder text for when nothing is selected. |
 
+## Testing with ember-hook
+The select component is accessible using ember-hook with the top level hook name or you can access the internal components as well -
+* Input field hook - `$hook('<hook-name>-input')`
+* List hook - `$hook('<hook-name>-list').find('li')`
```
Comment (sglanzer-fire) [inline docs/frost-select.md]: Do we need `li` here?  The hook was on the `ul` which seems correct

**Comment by sglanzer-fire** [comment]
Awesome - ember-hook is seriously the best testing tool since sliced bread ;)

On a side note - is it possible for Reviewable to not be listed in the Changelog section (since it currently ends up in the actual changelog, which seems incorrect)

**Comment by sglanzer-fire** [comment]
Approved - just check over the `ul` comment

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/164/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

File: docs/frost-select.md (hunk 1 above)
Comment (juwara0) [inline docs/frost-select.md]: I agree. The hook is for the list. The `<li>` has been removed.

**Comment by juwara0** [comment]
👍  self approving after addressing the comment to modify the documentation item about how to access the hook for the list.

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/164/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

**Pull request merged**